R-package/R/lgb.importance.R

```diff
@@ -1,5 +1,5 @@
 #' @name lgb.importance
-#' @title Compute feature importance in a model
+#' @title Compute feature importance in a model (testing)
```

Comment:
I added this as an example of the type of change that should trigger a failure. I'll remove it once we see if the new task works as expected.

Reply:
it  looks like this worked as expected!

https://github.com/microsoft/LightGBM/pull/3205/checks?check_run_id=836077809

![image](https://user-images.githubusercontent.com/7608904/86504872-6faa0400-bd83-11ea-8e2c-fef959d63736.png)